tests/utils/log/test_logging.py

```diff
@@ -21,72 +21,84 @@
 class TestS3TaskHandler(unittest.TestCase):
 
     def setUp(self):
-        super(S3TaskHandler, self).setUp()
+        super(TestS3TaskHandler, self).setUp()
         self.remote_log_location = 'remote/log/location'
+        self.local_log_location = 'local/log/location'
+        self.s3_log_location = 's3/log/location'
+        self.filename_template = ''
         self.hook_patcher = mock.patch("airflow.hooks.S3_hook.S3Hook")
         self.hook_mock = self.hook_patcher.start()
         self.hook_inst_mock = self.hook_mock.return_value
         self.hook_key_mock = self.hook_inst_mock.get_key.return_value
         self.hook_key_mock.get_contents_as_string.return_value.decode.\
             return_value = 'content'
+        class UnitTestS3TaskHandler(S3TaskHandler):
+            def test_instance(self):
+                return True
```

Comment:
What is this subclass and method for? It doesn't appear to be used anywhere in the codebase.

Reply:
(the test_instance method isn't used that I can see, the class is just used in here.)

Reply:
Ah yea - I copied from another test I was building but turns out I didn't need it, thanks for pointing it out